prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -165,6 +170,20 @@ public async Task SendRemoteUsersUpdatedAsync(Site site)
             await Send(PRIME_EMAIL, new[] { MOH_EMAIL, PRIME_SUPPORT_EMAIL }, subject, body, attachments);
         }
 
+        public async Task SendRemoteUsersNotificationAsync(Site site, IEnumerable<RemoteUser> remoteUsers)
+        {
+            var subject = "Remote Practitionner Notification";
```

Comment:
"Practitioner"

Reply:
![image](https://user-images.githubusercontent.com/53796823/92145077-88a75380-edcc-11ea-8ce9-968e6c66d1a1.png)
Thats how it is spelled in the ticket, not sure if intentionally.

Reply:
oh dear 😅

Reply:
Confirmed definitely not spelled correctly